### DISCUSSION TITLE
DIYRuZ_R4_5 documentation

### DISCUSSION THREAD

**Discussion by fedegrc** [Koenkk/zigbee2mqtt#9504] [Devices]
Hi, I am a new user. I have a DIYRuZ_R4_5 that I can control without problem, but in a different way than the documentation.

The doc in [Switch (bottom_left endpoint)](https://www.zigbee2mqtt.io/devices/DIYRuZ_R4_5.html#switch-bottom-left-endpoint) say:

> To control this switch publish a message to topic zigbee2mqtt/FRIENDLY_NAME/set with payload {"state_bottom_left": "ON"}, {"state_bottom_left": "OFF"} or {"state_bottom_left": "TOGGLE"}.

Using `mosquitto_pub` for testing, the following works fine, turning ON/OFF the buzzer:
```shell
-t zigbee2mqtt/DIYRuZ_R4/set -m '{"state_center": "TOGGLE"}'
```

but this doesn't work (it should change the relay):
```shell
-t zigbee2mqtt/DIYRuZ_R4/set -m '{"state_bottom_left": "TOGGLE"}'
```
producing this log en z2m:
```shell
error 2021-11-04 17:41:49: Device 'DIYRuZ_R4' has no endpoint 'left'
```
I'm currently using the following format to control the buzzer and switches, which works find (note I removed `state_`):
```shell
-t zigbee2mqtt/DIYRuZ_R4/center/set -m 'TOGGLE'
-t zigbee2mqtt/DIYRuZ_R4/bottom_left/set -m 'TOGGLE'
```

am I doing something wrong or the doc should be updated?

**Comment by kirovilya** [comment]
you can test the fix in the latest zigbee2mqtt dev branch

**Reply by fedegrc** [reply]
Tested with [koenkk/zigbee2mqtt:latest-dev](https://hub.docker.com/layers/koenkk/zigbee2mqtt/latest-dev/images/sha256-d90dcf4b3e4944950ca8e883c8c28df78e6d5dbfcc0881cbb356a5f3a374395d?context=explore). It works as expected now, thanks!
